adyenv6core/src/com/adyen/v6/factory/AdyenRequestFactory.java

```diff
@@ -181,10 +178,15 @@ else if (adyenPaymentMethod.indexOf(PAYMENT_METHOD_ONECLICK) == 0) {
                 }
             }
         }
-        //For alternate payment methods like iDeal, Paypal etc.
-        else {
-            updatePaymentRequestForAlternateMethod(paymentsRequest, cartData, customerModel);
-        }
+        else
+            //Set Boleto parameters
+            if (cartData.getAdyenPaymentMethod().indexOf(PAYMENT_METHOD_BOLETO) == 0) {
+                setBoletoData(paymentsRequest, cartData);
+            }
```

Comment:
shall we move this if to a elsif a level higher and then a else with the below else ?

Reply:
Can you please provide an example ?

Reply:
```suggestion
          else if (cartData.getAdyenPaymentMethod().indexOf(PAYMENT_METHOD_BOLETO) == 0) {
            setBoletoData(paymentsRequest, cartData);
          }
          else {
	    updatePaymentRequestForAlternateMethod(paymentsRequest, cartData, customerModel);
	     }
```
So the _else if_ on the same level of the previous one(the oneclick) and the else with updatePaymentRequestForAlternateMethod outside like it was before

Reply:
I think it was looking one level lower because of alignment. I have updated formatting.

Reply:
Yes perfect!